### PR TITLE
Done using marco's PPA, :sail: bon voyage!

### PR DESCRIPTION
**Pull request by lazypower**
Revert "Move to using Marco's PPA to test the charm2/charm-tools overlap"

This reverts commit eca10a27080ad4b09ea9a0296e1a63f6c8eeddda.

### CODE REVIEW

File: install-review-tools.sh [hunk 1]
```diff
@@ -2,8 +2,7 @@
 set -e
 HOME=/home/ubuntu
 
-# Use marco's ppa while we test the charm/charm-tools overlap stuff
-sudo add-apt-repository ppa:marcoceppi/charm-tools-2.0
+sudo add-apt-repository ppa:yellow/ppa
```
Comment (mbruzek) [inline install-review-tools.sh]: Don't you need a -y for add-apt-repository?

I see that the old version did not use it, but I wonder if that would come back to bite us later.

Comment (marcoceppi) [inline install-review-tools.sh]: why do you need yellow/ppa?

Comment (marcoceppi) [inline install-review-tools.sh]: @mbruzek it's not necessary. add-apt-repository can detect when you're not in an interactive shell and just assumes `-y` not bad to have though

Comment (marcoceppi) [inline install-review-tools.sh]: @chuckbutler to clarify, you'll need ppa:juju/devel for latest charm/charm-tools

**Pull request merged**